core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -494,12 +494,12 @@ public DDF sort(List<String> columns, List<Boolean> ascending, Boolean inPlace)
   }
 
   @Override public DDF castType(String column, String newType, Boolean inPlace) throws DDFException {
-    if (inPlace) {
-      DDF ddf = castType(column, newType);
-      return this.getDDF().updateInplace(ddf);
-    } else {
-      return castType(column, newType);
-    }
+    throw new UnsupportedOperationException();
```

Comment:
Before this change core DDF also didn't support casttype, so should be fine merging this. But I'll do the jdbc implementation part in this sprint.